ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -74,9 +74,10 @@ Item {
         id: adHocChatButton
         icon.name: "edit"
         objectName: "startChatButton"
+        icon.color: Theme.palette.directColor1
```

Comment:
You mean this, @caybro ??

Reply:
Yes, that's exactly what I meant :D sry, need coffee